### ISSUE TITLE
Add Webapp Manager

### ISSUE THREAD

**Issue by Zahrun** [NixOS/nixpkgs#212089]
**Project description**
Run websites as if they were apps

**Metadata**

* homepage URL: http://packages.linuxmint.com/pool/main/w/webapp-manager/
* source URL: https://github.com/linuxmint/webapp-manager
* license: gpl3
* platforms: linux

**Comment by spinus** [comment]
Just checked the code, it contains so many hardcoded filesystem paths in the python code and scripts that will be some work to patch all of that stuff.

**Comment by Zahrun** [comment]
Hi @spinus, thank you for checking the code. Maybe https://github.com/NixOS/nixpkgs/issues/183095 would be easier to implement? Webapp Manager is nice though since it supports multiple browsers, not just electron, and has options for having navigation bar and stuff.

**Comment by spinus** [comment]
If you want to avoid electron, and patching this is too much work, I guess that app is really not doing that much over all. You can create desktop files with browser profiles yourself https://github.com/linuxmint/webapp-manager/blob/aab427a9d4ce96a47f90f22a552ea37fda5ca970/usr/lib/webapp-manager/common.py#L264
But in that case it's more or less electron in that sense, that each "app" is separate process and memory, so quite big things anyway.

Check if firefox app containers it's not enough for what you are trying to do. I use app containers + extension to automatically switch per domain and kinda works.

**Comment by Zahrun** [comment]
Right, creating desktop files with browser profiles myself works for me, thank you for the suggestion and the link to the code.
GUI would be nice but would requires a heavy patch.
